### PR TITLE
Center description in readme

### PR DESCRIPTION
**Pull request by paescuj**
Seems to fit a little better since all other elements in the header are centered as well

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Tridactyl
 <br>
 </h1>
 
-<h4>Replace Firefox's default control mechanism with one modelled on the one true editor, Vim. </h4>
+<h4 align="center">Replace Firefox's default control mechanism with one modelled on the one true editor, Vim.</h4>
```
Comment (paescuj) [inline readme.md:9]:
```suggestion
<h4 align="center">Replace Firefox's default control mechanism with one modelled on the one true editor, Vim.</h4>
```

**Comment by paescuj** [comment]
Should we skip e2e tests on readme changes? Could be achieved with the [`paths-ignore`](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-excluding-paths) option:

```yaml
    paths-ignore:
      - 'readme.md'
      # or maybe even
      - '**.md'
```

**Comment by bovine3dom** [comment]
Thanks for the PR. I need to just check how it looks on my PC.

As for the markdown files, some of them are compiled to HTML in the build process so we shouldn't exclude all of them. We could exclude the readme but I don't think it's a problem that arises often.

The concurrency cancelling seems like a good idea :)

**Comment by paescuj** [comment]
Thanks!

> As for the markdown files, some of them are compiled to HTML in the build process so we shouldn't exclude all of them. We could exclude the readme but I don't think it's a problem that arises often.

Makes sense! However, since there are only two additional lines, I'd do the filtering for the `readme.md` file.

> The concurrency cancelling seems like a good idea :)

Great! I'm going to create a pull request for this 👍

**Pull request merged**